operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -45,17 +47,42 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
       ```
 
   > **NOTE:** All subsequent steps of this procedure should be performed within the `/root/site-init` directory created in this step.
-   
-2. Update the LDAP settings in the customizations.yaml file.
 
-   1. (Optional) Add the LDAP CA certificate in the certs.jks section of customizations.yaml.
+2. Repopulate the keycloak_users_localize and cray-keycloak Sealed Secrets in the customizations.yaml file with the desired configuration.
 
-      Follow step 8 under the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure.
-   
-   2. Update the LDAP settings.
-  
-      LDAP connection information is stored in the keycloak-users-localize Secret in the services namespace. 
-      In the customizations.yaml file, set the values for the keycloak_users_localize keys in the spec.kubernetes.sealed_secrets field:
+   1. Check to see if the `generate:` sections of the Sealed Secrets have been populated with encrypted Sealed Secrets already:
+      
+      ```bash
+      ncn-m001# yq read ./customizations.yaml spec.kubernetes.sealed_secrets.keycloak_users_localize
+      apiVersion: bitnami.com/v1alpha1
+      kind: SealedSecret
+      metadata:
+        annotations:
+          sealedsecrets.bitnami.com/cluster-wide: "true"
+        creationTimestamp: null
+        name: keycloak-users-localize
+      spec:
+        encryptedData:
+          ldap_connection_url: AgAmdO19GaLs3Yr9apnJ/JDuQS+6yMC+LlZrPO8g+9UvF2+0X1TifH/bPb0Dw4VMN/2MURx/vvwJE2DTz9yajuW3YJEdwD4o6z/OZ/qLDxu2u+HZSwRnWLWK6ROTBGMP7r0zOdQIDoeeAZw03+d4/UmiBlTlJhl+DJzcS8VbfJV+neNcZ0p5zcM7skqI5NL4teNItHoeuITC2QQ+TRQc/XOrkj3JxvrzFEtEstJz8fXOUXBOwakhRRzUZl9aAYcT6raK3mPQDg14AkM4JVCeku+h6O4OOoOIygC8FzfrXy+LWE93UZjw/0ZM+c6bqOLd7odGto5EylLaV7HS9V8trUPfKExBbYqoRzm+IU9eG3k8Gr7ijT9hRhr1wiV73DFy5NnNB0uAjFClnPXbqntbnSScLxHgFqrnitKM19RzcuHFaZ0Hq0S0VPO8az8BL7jiCkhlxqT0WN6I5RerHPt0PocikKJ/S58a2dc8uGwMgyAPCcJwNXnh7qoA3pgL72kD292ReCYUz6XR51XQAaW1S5O/OaI3VKCCHZAw+qCgW3tCraL3mjzuTYSQQQsDFicxcgQZVz26S/9ATW+3g8btOZlsJ8aA9zpSGAf+uo2N/OKDCmU60fxFTvddcYDlWZ9ZXM1q9lnDTzz1T+QMCZ/f+c2El4CgNvrKJvLueduuagGNfrYoVgGoQ76e/WVi866d1a0=
+        template:
+          metadata:
+            annotations:
+              sealedsecrets.bitnami.com/cluster-wide: "true"
+            creationTimestamp: null
+            name: keycloak-users-localize
+      generate:
+        data: {}
+      ```
+
+      If these sections were populated during the install, proceed to the next sub-step to 
```

Comment:
This says that only if these sections were populated during the install should the admin proceed to the next sub-step. The next sub-step is "Update the LDAP settings with the desired configuration." where the admin configures keycloak-users-localize with all their settings for their LDAP server (after removing the existing sealed secrets). The admin always has to configure keycloak-users-localize with their settings for their LDAP server, it's not optional.